src/components/dialogs/searchDialog/PluginsManager.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <v-dialog
+    v-model="searchEngineState.isPluginManagerOpen"
+    max-width="20rem"
+    scrollable
+  >
+    <v-card>
+      <v-card-title> <v-icon>mdi-toy-brick</v-icon> Plugins manager </v-card-title>
+      <v-card-text>
+        <v-switch
+          v-for="(plugin, key) in searchEngineState.searchPlugins"
+          :key="key"
+          :false-value="false"
+          :input-value="plugin.enabled"
+          :value="plugin.enabled"
```

Comment:
Why those lines?

Reply:
I believe you are asking here `why` I do not use `v-model`? 

`v-model` mutates the state. The reason for using these lines is that they continue to work as `v-model` but do not mutate the state. Just following one of the main concepts of redux pattern. (Only a reducer can mutate the state)

P.S. Actually when did this `switch` I found out we mutate the state that way without using the reducers. Based on it I understand I did mutate the state using `v-model` on several other places so I will have to change that.

Reply:
Just use `input-value`, remove `false-value` and `value` is enough.

Reply:
You are right, I already do the handling in the actions. Good catch!

Reply:
And you will found the event value will be `false` now.